react/features/welcome/functions.native.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import React from 'react';
+
+import { getFeatureFlag, WELCOME_PAGE_ENABLED } from '../base/flags';
+import { IconArrowBack } from '../base/icons';
+import HeaderNavigationButton
+    from '../conference/components/native/HeaderNavigationButton';
+
+/**
+ * Determines whether the {@code WelcomePage} is enabled by the app itself
+ * (e.g. Programmatically via the Jitsi Meet SDK for Android and iOS). Not to be
+ * confused with {@link isWelcomePageUserEnabled}.
+ *
+ * @param {Function|Object} stateful - The redux state or {@link getState}
+ * function.
+ * @returns {boolean} If the {@code WelcomePage} is enabled by the app, then
+ * {@code true}; otherwise, {@code false}.
+ */
+export function isWelcomePageAppEnabled(stateful: Function | Object) {
+    if (navigator.product === 'ReactNative') {
```

Comment:
Remove this if, since now this file is only executed on mobile.

Reply:
Don't I need this check to see if welcome page is available?

Reply:
Yes, but the condition will always be true `if (navigator.product === 'ReactNative') {` since this function is only used in native.

Reply:
Right. I removed this condition and kept the featureFlag check.